packages/global/src/utils/math.ts

```diff
@@ -26,3 +26,45 @@ export const clamp = (value: number, min: number, max: number): number => {
   }
   return value;
 };
+
+/**
+ * Converts an angle from degrees to radians.
+ *
+ * @param {number} angle - The angle in degrees.
+ * @returns {number} - The angle in radians.
+ */
+export function toRadians(angle: number) {
```

Comment:
This global math module only exists for historical reasons. Adding the utils within the edgeless related dirs should be the better scope.